reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md

```diff
@@ -90,19 +91,13 @@ extension **.t7f** is uncommon, but the hexadecimal output **%PDF** shows that i
 
 ### -Encoding
 
-Specifies the type of encoding for the target file. The default value is **ASCII**.
+Specifies the type of encoding for the target file. The default value is **Default**.
```

Comment:
Ensure acceptable values match in YAML to new list and update default value.

Reply:
fixed